lib/runtime/test_helpers.go

```diff
@@ -161,3 +169,123 @@ func RemoveFiles(files []string) error {
 	}
 	return nil
 }
+
+// CreateTestExtrinsic builds a new extrinsic using centrifuge pkg
+func CreateTestExtrinsic(t *testing.T, rt Instance, genHash, blockHash common.Hash, nonce uint64, call string, args ...interface{}) string { //nolint
```

Comment:
```suggestion
func newTestExtrinsic(t *testing.T, rt Instance, genHash, blockHash common.Hash, nonce uint64, call string, args ...interface{}) string { //nolint
```

Reply:
Updated to `NewTestExtrinsic` as it needs to be exported